SPECS/containerized-data-importer/containerized-data-importer.spec

```
@@ -0,0 +1,522 @@
+#
+# spec file for package containerized-data-importer
+#
+# Copyright (c) 2022 SUSE LLC
+#
+# All modifications and additions to the file contributed by third parties
+# remain the property of their copyright owners, unless otherwise agreed
+# upon. The license for this file, and modifications and additions to the
+# file, is the same license as for the pristine package itself (unless the
+# license for the pristine package is not an Open Source License, in which
+# case the license is the MIT License). An "Open Source License" is a
+# license that conforms to the Open Source Definition (Version 1.9)
+# published by the Open Source Initiative.
+
+# Please submit bugfixes or comments via https://bugs.opensuse.org/
+#
+
+Summary:        Container native virtualization
+Name:           containerized-data-importer
+Version:        1.51.0
+Release:        1%{?dist}
+License:        Apache-2.0
```

Please use the ["Short Name" from Fedora's guidelines](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses).

```suggestion
License:        ASL 2.0
```

---

This is corrected.